SPECS/xapi.spec

```diff
@@ -73,6 +73,7 @@ Patch1007: xapi-23.31.0-xapi-service-depends-on-systemd-tmpfiles-setup.patch
 Patch1008: xapi-23.31.0-use-lib-guess-content-type.XCP-ng.patch
 # Remove when we get 24.14.0 release
 Patch1009: xapi-24.11.0-pci-passthrough.XCP-ng.patch
+Patch1010: xapi-24.11.0-disable-fileserver-option.XCP-ng.patch
```

Comment:
Better also add a comment stating in which release we can expect this patch in upstream XAPI

Reply:
It's the same release as above 24.14 so ithought it applieed to the whole batch until there's another comment :thinking:

Reply:
Oh, right. All good then.